pkg/cinderbackup/volumes.go

```diff
@@ -1,16 +1,28 @@
 package cinderbackup
 
 import (
+	"strconv"
+
 	cinderv1beta1 "github.com/openstack-k8s-operators/cinder-operator/api/v1beta1"
 	"github.com/openstack-k8s-operators/cinder-operator/pkg/cinder"
 	corev1 "k8s.io/api/core/v1"
 )
 
 // GetVolumes -
-func GetVolumes(parentName string, name string, extraVol []cinderv1beta1.CinderExtraVolMounts) []corev1.Volume {
+func GetVolumes(parentName string, name string, secretNames []string, extraVol []cinderv1beta1.CinderExtraVolMounts) []corev1.Volume {
 	var config0640AccessMode int32 = 0640
+	var dirOrCreate = corev1.HostPathDirectoryOrCreate
 
-	backupVolumes := []corev1.Volume{
+	volumes := []corev1.Volume{
+		{
+			Name: "var-lib-cinder",
+			VolumeSource: corev1.VolumeSource{
+				HostPath: &corev1.HostPathVolumeSource{
+					Path: "/var/lib/cinder",
```

Comment:
I know we have this on the cinder-volume service, but why is this necessary in Operator deployed cinder?
Is it for NFS mounts?

Reply:
Usually I wouldn't want to add this change in this patch, but since I suspect we need it for NFS when volume and backup are running in the same host we can leave it to be the same as TripleO and then evaluate if we can remove it.